docs/basics/101-101-create.rst

```diff
@@ -67,7 +67,7 @@ to as the history of a dataset.
 
 This history is almost as small as it can be at the current state, but let's take
 a look at it. For looking at the history, the code examples will use :command:`git log`,
-a built-in git command [#f1]_.
+a built-in git command [#f1]_. You can get out of git log by pressing q.
```

Comment:
```suggestion
a built-in git command [#f1]_. Your git log *might* be opened in a `terminal pager <https://en.wikipedia.org/wiki/Terminal_pager>`_ that lets you scroll up and down with your arrow keys, but not enter any more commands. If this happens, you can get out of git log by pressing q.
```

Reply:
Exactly, on my laptop I had to press q to get out of it (even at this point, which led me to the comment). I like your added explanation!

Reply:
Good to know, now we just need to get it in there, and have you as the person committing it to show up in the contributors graph of this repository (and not only the README) as well :)

Reply:
Awesome, branch is restored! You should be able to click on "commit suggestion" under my first comment